core/src/main/scala/chisel3/internal/MonoConnect.scala

```diff
@@ -178,6 +188,35 @@ private[chisel3] object MonoConnect {
       case (sink, source) => throw MismatchedException(sink.toString, source.toString)
     }
 
+  /** Check whether two [[Vec]]s can be bulk connected (<-) in FIRRTL. */
```

Comment:
```suggestion
  /** Check whether two [[Vec]]s can be bulk connected (<=) in FIRRTL. */
```
`<-` is the unsafe (stringly-typed) connect in FIRRTL. `<=` is the stricter form. They are both bidirectional bulk connects. We need to emit `<=` from `import chisel3._`

Reply:
Looks like `<=` is emitted by `chisel3.internal.firrtl.Connect`, and `<-` by `chisel3.internal.firrtl.BulkConnect`: https://github.com/chipsalliance/chisel3/blob/master/src/main/scala/chisel3/internal/firrtl/Emitter.scala#L74

So I want to be calling `chisel3.internal.firrtl.Connect`? Or should I be changing the behavior of `chisel3.internal.firrtl.BulkConnect`?

Reply:
the `chisel3.internal.firrtl.*Connect` names are misnomers, both `<=` and `<-` are bulk connects, `<-` is called "PartialConnect" in FIRRTL

Reply:
Do not change the behavior of `BulkConnect` (since that is what is issued by `import Chisel._`), instead you can just issue `chisel3.internal.firrtl.Connect`